Packs/PcapAnalysis/Scripts/PcapFileExtractStreams/PcapFileExtractStreams_test.py

```diff
@@ -2,6 +2,31 @@
 import json
 
 
+def equals_object(obj1, obj2) -> bool:
+    if type(obj1) != type(obj2):
+        return False
+    elif isinstance(obj1, dict):
+        for k1, v1 in obj1.items():
+            if k1 not in obj2:
+                return False
+            if not equals_object(v1, obj2[k1]):
+                return False
+        return not (set(obj1.keys()) ^ set(obj2.keys()))
+    elif isinstance(obj1, list):
+        # Compare lists (ignore order)
+        list2 = list(obj2)

```

Comment:
```suggestion

```
obj2 should already be a list because obj1 is a list and obj2 should be the smae as obj1 because of the condition in line 6

Reply:
@MosheEichler `list2 = list(obj2)` is a code to copy from `obj2`. At line 21, `list2` is modified, but it doesn't affect to `obj2`. That's the code in order not to modify `obj2` given from args.